awscli/customizations/s3/subcommands.py

```diff
@@ -392,6 +392,15 @@
 }
 
 
+FORCE_GLACIER_TRANSFER = {
```

Comment:
I'm not crazy about `--force-glacier` because it isn't clear what you're forcing. `--force-glacier-transer` is slightly clearer. Even then, maybe something like `--force-glacier-attempt` would be better because it is clear that you are forcing the CLI to make an attempt with glacier.